src/helics/application_api/helicsPrimaryTypes.cpp

```diff
@@ -34,6 +34,13 @@ static const std::set<std::string> falseString{"0",
                                                "false",
                                                "False",
                                                "FALSE",
+                                               "OFF",
+                                               "Disabled",
+                                               "disable",
+                                               "disabled",
+                                               "DISABLE",
+                                               "off"
+                                               "DISABLED",
```

Comment:
```suggestion
                                               "off",
                                               "Off",
                                               "OFF",
                                               "disabled",
                                               "Disabled",
                                               "DISABLED",
                                               "disable",
                                               "Disable",
                                               "DISABLE",
```
Include all capitalization variants, and sort so it is easier to tell what is there/missing.